src/main/k/working/tiny/specs/a-to-c/Makefile

```diff
@@ -0,0 +1,9 @@
+DEFINITION_NAME := ../../tiny
+
+include $(CURDIR)/../../../include.mk
+
+SPEC_FILE := a-to-c-spec.k
+
+repl-test: scripts/automatic.repl.test scripts/manual.repl.test
+
+repl-golden: scripts/automatic.repl.output.golden scripts/manual.repl.output.golden
```

Comment:
Add the target:

```
test-k: repl-test
```

This _should_ make it run on PRs as well. We need to double-check in the jenkins output that it actually does.